node-graph/gcore/src/graphic_element/renderer.rs

```diff
@@ -20,22 +20,43 @@ use std::fmt::Write;
 #[cfg(feature = "vello")]
 use vello::*;
 
+#[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
+pub enum ClickTargetGroup {
+	Subpath(bezier_rs::Subpath<PointId>),
+	PointGroup(ManipulatorGroup<PointId>),
```

Comment:
Why is manipulator group now a point group? Please avoid introducing redundant terminology.